config/deploy.yml

```diff
@@ -0,0 +1,30 @@
+service: ruby-europe
+image: rubyeurope/rubyeurope.com
+
+servers:
+  web:
+    hosts:
+      - 64.227.117.90
+proxy:
+  app_port: 3000
+  ssl: true
+  host: do.rubyeurope.com
+  healthcheck:
+    interval: 2
+
+registry:
+  server: ghcr.io
+  username: KAMAL_REGISTRY_USERNAME
+  password:
+    - KAMAL_REGISTRY_PASSWORD
+
+builder:
+  arch: amd64
+
+env:
+  clear:
+    WEB_CONCURRENCY: 0
+    RAILS_ENV: production
+    HOST: do.rubyeurope.com
```

Comment:
since we're merging to `main`, it should probably be just rubyeurope.com as well?